### ISSUE TITLE
Renders everything as pink on Mac (with windows target)

### ISSUE THREAD

**Issue by capitaljmedia** [ValveSoftware/the_lab_renderer#4]
When a project with the lab renderer is opened on a mac unity editor, anything using the steam material is pink as if the material is missing. Even if the build target is set to windows 64bit. This makes it impossible for team members with macs to work on non-vr parts of the game (level design, adding models, scripted events, etc..).

**Comment by SmartCarrion** [comment]
I second this one...glad it wasn't just something I did wrong!

**Comment by eastes** [comment]
This happened to me after building on PC, for PC.

It was because I had a mesh with material that was set to Additive. Changed to Alpha Blend and it worked.

**Comment by capitaljmedia** [comment]
@eastes do you mean that after you changed that one material to alpha blend that all your other pink materials were restored? or was only that one material pink?

**Comment by eastes** [comment]
@capitaljmedia Yes, after I changed that one material, my builds were working again for all materials. That one material was only on one object in my scene. That one object wasn't even enabled. Just to confirm, the pink problem only happened in my builds. The editor was fine.

**Comment by RosaryMala** [comment]
None of the lab renderer shaders support OpenGL. 

Removing `#pragma only_renderers d3d11` from the shaders allows them to at least show up in the editor when you're on OpenGL, but it causes the shader to handle lightmaps incorrectly, showing them over-bright. The system also quickly runs out of GPU memory.
